config/routes.rb

```diff
@@ -50,6 +50,7 @@
         mount flipper_ui, at: "feature_flags"
 
         resources :data_update_scripts, only: [:index]
+        post "/run_data_update_scripts", to: "data_update_scripts#run_data_update_scripts"
```

Comment:
as the URL only run a single script it should probably be singular. Even though @citizen428 would say here to use RESTy routes on a different controller I think we can add the extra route here but use `resources` properly using [member routes](https://guides.rubyonrails.org/routing.html#adding-member-routes). Something like: `POST /data_update_scripts/{id}/force-run` or something like that